server/webhook_parser.go

```diff
@@ -272,12 +279,15 @@ func parseWebhookCommentDeleted(jwh *JiraWebhook) (Webhook, error) {
 }
 
 func parseWebhookCommentUpdated(jwh *JiraWebhook) Webhook {
-	return &webhook{
+	wh := &webhook{
 		JiraWebhook: jwh,
 		eventTypes:  NewStringSet(eventUpdatedComment),
```

Comment:
totally OT, but shouldn't this just be, `events`? 0/5 would read better. Just asking for an opinion, not a change.

Reply:
You mean, instead of `wh`, the variable should be `events`? But there's only ever able to be one 'event'. So maybe event?

Reply:
No, sorry GH put it out of context, I was commenting on the "white" existing code. NM.